packages/toolkit/src/view/knowledge/components/KnowledgeBaseTab.tsx

```diff
@@ -0,0 +1,135 @@
+import { Separator, Skeleton } from "@instill-ai/design-system";
+import * as React from "react";
+import { KnowledgeBaseCard } from "./KnowledgeBaseCard";
+import { CreateKnowledgeBaseCard } from "./CreateKnowledgeBaseCard";
+import { CreateKnowledgeDialog } from "./CreateKnowledgeDialog";
+import {
+  useGetKnowledgeBases,
+  useCreateKnowledgeBase,
+} from "../../../lib/react-query-service/knowledge";
+import {
+  InstillStore,
+  useAuthenticatedUser,
+  useInstillStore,
+  useShallow,
+} from "../../../lib";
+import { KnowledgeBase } from "../../../lib/vdp-sdk/knowledge/types";
+
+type KnowledgeBaseTabProps = {
+  onKnowledgeBaseSelect: (knowledgeBase: KnowledgeBase) => void;
+  onDeleteKnowledgeBase: (knowledgeBase: KnowledgeBase) => void;
+  accessToken: string | null;
+};
+
+export const KnowledgeBaseTab = ({
+  onKnowledgeBaseSelect,
+  //   onDeleteKnowledgeBase,
```

Comment:
Should this be removed?

Reply:
No, as You can see in future iteration the delete confirmation popup will be across tabs so started working in that direction